.changeset/large-dolphins-roll.md

```diff
@@ -0,0 +1,29 @@
+---
+'@astrojs/markdown-remark': minor
+'astro': minor
+---
+
+Adds a new configuration option for Markdown syntax highlighting `excludeLangs` 
+
+This option provides better support for diagramming tools that rely on Markdown code blocks, such as Mermaid.js and D2 by allowing you to exclude specific languages from Astro's default syntax highlighting.
+
+This option allows you to avoid rendering conflicts with tools that depend on the code not being highlighted without forcing you to disable syntax highlighting for other code blocks.
+
+The default value for `excludeLangs` is `['math']` and remains unchanged by default in this release.
+But users can now override it to exclude other languages or exclude no languages.
```

Comment:
```suggestion
```
I don't think this is necessary here. This doesn't have to be docs, but more an introduction to let people know about the new feature. Adding too many details can detract from just giving people the main idea so they can decide if this is something they care about/want to read more about.